python-phoenixdb/phoenixdb/types.py

```diff
@@ -222,9 +225,9 @@ def __cmp__(self, other):
     (2011, common_pb2.STRING),  # NCLOB
     (2009, common_pb2.STRING),  # SQLXML
     # Returned by Avatica for Arrays in EMPTY resultsets
-    (2000, common_pb2.BYTE_STRING)  # JAVA_OBJECT
+    (2000, common_pb2.BYTE_STRING),  # JAVA_OBJECT
     # These are defined by JDBC, but cannot be mapped
-    # NULL
+    (0, common_pb2.NULL) # NULL
```

Comment:
The comment is obviously not true for this one.
Please move this to the start (in numerical order)

Reply:
They are not numerically ordered.
Still, it should be grouped with the other standard Phoenix types.